src/test/unit/StrategyBaseTVLLimitsUnit.sol

```diff
@@ -116,10 +128,43 @@ contract StrategyBaseTVLLimitsUnitTests is StrategyBaseUnitTests {
         require(strategyWithTVLLimits.totalShares() == depositAmount + sharesBefore, "total shares not updated correctly");
     }
 
+    function testDepositTVLLimit_ThenChangeTVLLimit(uint256 maxDeposits, uint256 newMaxDeposits) public {
+        cheats.assume(maxDeposits > 0);
```

Comment:
again this actually seems unnecessary?

Reply:
fixed https://github.com/Layr-Labs/eigenlayer-contracts/pull/45/commits/09fd1573545db8d448b6e2f64f9c07da4795c137